North-Seattle-College/ad440-winter2021-thursday-repo
# Implemented react pagination for /users

This PR Resolves #256 

This PR holds a script that implements React paging for  /users 

**Testing**

1. Navigate to _ui/react-ui_ folder
2. Download packages and dependencies required to run the app using `npm install`
3. Start the React app by typing `npm start`
4. When the app is successfully running on your localhost, please press the _/users_ button or type http://localhost:3000/users (assuming you hosted the app on localhost:3000) on your URL bar
5. A paginated page that shows the list of users displays and should look like the picture below
6. Scroll through pages by using the next buttons or clicking the page number

![Screen Shot 2021-03-09 at 10 39 58 PM](https://user-images.githubusercontent.com/44041437/110587355-6bab5e80-8128-11eb-8803-48293cff9b9e.png)


DATE | ACTIVITY | TIME
-- | -- | --
03/07 | Research on the best way to implement pagination |1 hour
03/09 | Writing a script for implementing pagination | 3 hour
